doc/dev/sharp_bits.rst

```diff
@@ -510,6 +510,7 @@ that doesn't work with Catalyst includes:
 
 - ``jax.numpy.polyfit``
```

Comment:
I thought Romain added support for `polyfit` via custom calls?

Reply:
Nice catch, I wasn't aware. If so we could remove this.

Reply:
Indeed! This works:

```
>>> x = jnp.array([1.2, 0.1, 2.7, 0.6])
>>> y = jnp.array([0.2, 0.4, 0.6, 0.8])
>>>
>>> @qml.qjit
... def fn(x):
...     return jnp.polyfit(x, y, deg=4)
...
>>> fn(x)
array([-0.0059191 ,  0.01930699,  0.01312885,  0.09407333, -0.22579063])
```

Reply:
I updated https://app.shortcut.com/xanaduai/epic/65875?cf_workflow=500000005&ct_workflow=all&group_by=none&vc_group_by=day